### PR TITLE
Readme turning off fees

### PR DESCRIPTION
**Pull request by Sriep**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ Set up a network called testnet0 for each of these node containers to talk to ea
 
   1.4) If you want to adjust the network relay time, set the value of `network.relay_time`
 
+  1.5) If you want to turn off fees adjust `server_chain.smart_contract.mine` from `true` to `false`
```
Comment (peterlimg) [inline README.md]: Should be `miner`

File: code/go/0chain.net/smartcontract/storagesc/README.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ Provide
 
 The transaction also updates blobber's 'last health check' making it healthy.
 
+The blobber will need enough tokens in its operational wallet to pay the fee for registration transaction.
+If that is a problem fees can be switched off by adjusting
+the `server_chain.smart_contract.mine` setting in 0chain.yaml from `true` to `false`.
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/README.md]: Same as above

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
LGTM

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | staging  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | readme_fees  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request merged**